install/scripts/csm_services/steps/1.initialize_bootstrap_registry.yaml

```diff
@@ -62,23 +62,9 @@ spec:
           podman tag skopeo:csm-{{ getEnv "CSM_RELEASE" }} quay.io/skopeo/stable:latest
 
           podman run --rm --network host -v {{ getEnv "CSM_PATH" }}/docker:/images:ro \
-              quay.io/skopeo/stable sync --scoped --src dir --dest docker --dest-tls-verify=false --dest-creds admin:admin123 \
+              quay.io/skopeo/stable sync --all --scoped --src dir --dest docker --dest-tls-verify=false --dest-creds admin:admin123 \
```

Comment:
This step actually duplicates upload, performed by [setup-nexus.sh script](https://github.com/Cray-HPE/node-images/blob/main/metal-provision/scripts/nexus/setup-nexus.sh#L905-L923) for metal nexus, so I'm am not sure if we need it at all. But if we do upload, we must include `--all` option. Otherwise, skopeo only uploads `amd64` manifest without signature, which overrides original multi-platform manifest and detaches existing signature.